apps/frontend/src/constants/currencies.ts

```diff
@@ -2,6 +2,8 @@ export const USD = 'USD';
 
 export const BITCOIN = 'BTC';
 export const SOV = 'SOV';
+export const VP = 'VP';
+export const APR = 'APR';
```

Comment:
These definitely don't belong to `currencies.ts`. Move them to staking constants.